### DISCUSSION TITLE
Report misappropriation

### DISCUSSION THREAD

**Discussion by dongzirui542** [vectorwing/FarmersDelight#746] [General]
The mod was stolen by MCLans to NetEase Minecraft for a fee of 200 diamonds [about 2 CNY or 0.28 USD] and unauthorized
Unauthorized mod links: https://mclans.cn/?dt_portfolio=book
[link.zip](https://github.com/vectorwing/FarmersDelight/files/11607756/link.zip)

**Comment by vectorwing** [comment]
Thank you for reporting this to me!

Sadly, I have no idea if I can even do much about this. The project intently wraps itself in a standalone service, blatantly stealing my content (and likely other mods) in it, so there's no "website" to report to. :(
